src/endpoints/Account.ts

```diff
@@ -2,60 +2,46 @@ import Http from '../Http'
 import { IAccount } from '../interfaces/Account'
 import { IToken } from '../interfaces/Token'
 import { Routes } from '../routes'
+import { GET } from '../constants'
 
 export default class Account extends Http {
-  public async accountInfo(token: IToken): Promise<IAccount> {
+  public async accountInfo(token: IToken): Promise<IAccount | Error> {
```

Comment:
Rejected promise value cannot be typed, it's hardcoded to `any`, so adding `| Error` actually means a resolved promise can accept an Error value, which I don't think is ever true in out case: https://stackoverflow.com/questions/50071115/typescript-promise-rejection-type